docs/glassfish/publish/pom.xml

```diff
@@ -0,0 +1,239 @@
+                                    <excludes>META-INF/**</excludes>
+                                    <artifactItems>
+                                        <artifactItem>
+                                            <groupId>org.glassfish.docs</groupId>
+                                            <artifactId>distribution</artifactId>
+                                            <version>5.1.0</version>
+                                            <outputDirectory>
+                                                ${dir.510}
+                                            </outputDirectory>
+                                        </artifactItem>
+                                    </artifactItems>
+                                </configuration>
+                            </execution>
+                            <!--
+                                Add additional execution blocks here as
+                                new releases are made, using the above
+                                as a template.  Don't forget to update
+                                the docs.version property.
+                            -->
+                        </executions>
+                    </plugin>
+
+                    <!--
+                        Create a symlink from "latest" to the most recently
+                        released version of the docs.
+                    -->
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <version>1.8</version>
+                        <executions>
+                            <execution>
+                                <id>create-latest-link</id>
+                                <phase>process-resources</phase>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                                <configuration>
+                                    <target>
+                                        <delete
+                                          file="${site.output.dir}/docs/latest"
+                                          />
+                                        <symlink
+                                          link="${site.output.dir}/docs/latest"
+                                          resource="${docs.version}"/>
+                                    </target>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+
+                    <!--
+                        Publish the entire web site to the gh-pages branch.
+                    -->
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-scm-publish-plugin</artifactId>
+                        <version>3.0.0</version>
```

Comment:
Plugin version should be managed in the parent pom

Reply:
Moved version to top level pom.